client/components/paginationChooseLimit/index.tsx

```diff
@@ -14,6 +14,7 @@ export const PaginationLimit = ({ limit, setLimit, setPage }) => {
         Choose a value:
       </RLabel>
       <DropdownComponent
+        width=""
```

Comment:
we should make these optional props not just pass empty strings, if you are trying to fix type errors